SetupTemplate/SetupTemplate.wixproj

```diff
@@ -0,0 +1,57 @@
+    <Platform Condition=" '$(Platform)' == '' ">x86</Platform>
+    <ProductVersion>3.10</ProductVersion>
+    <ProjectGuid>ab5cea73-6f41-4811-b4ab-ce668f44e164</ProjectGuid>
+    <SchemaVersion>2.0</SchemaVersion>
+    <OutputName>$(output_name)</OutputName>
+    <OutputType>Package</OutputType>
+	<DefineSolutionProperties>false</DefineSolutionProperties>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|x86' ">
+    <OutputPath>bin\$(Configuration)\</OutputPath>
+    <IntermediateOutputPath>obj\$(Configuration)\</IntermediateOutputPath>
+    <DefineConstants>Debug</DefineConstants>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|x86' ">
+    <OutputPath>bin\$(Configuration)\</OutputPath>
+    <IntermediateOutputPath>obj\$(Configuration)\</IntermediateOutputPath>
+  </PropertyGroup>
+  <ItemGroup>
+    <WixExtension Include="WixUtilExtension">
+      <HintPath>$(WixExtDir)\WixUtilExtension.dll</HintPath>
+      <Name>WixUtilExtension</Name>
+    </WixExtension>
+    <WixExtension Include="WixUIExtension">
+      <HintPath>$(WixExtDir)\WixUIExtension.dll</HintPath>
+      <Name>WixUIExtension</Name>
+    </WixExtension>
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="Product.wxs" />
+  </ItemGroup>
+  <ItemGroup>
+    <Folder Include="Resources" />
+  </ItemGroup>
+  <ItemGroup>
+    <Content Include="Resources\black16x16.ico" />
+    <Content Include="Resources\black32x32.ico" />
+    <Content Include="Resources\License.rtf" />
+    <Content Include="Resources\InstBanner.bmp" />
+    <Content Include="Resources\InstDialog.bmp" />
+  </ItemGroup>
+  <Import Project="$(WixTargetsPath)" Condition=" '$(WixTargetsPath)' != '' " />
+  <Import Project="$(MSBuildExtensionsPath32)\Microsoft\WiX\v3.x\Wix.targets" Condition=" '$(WixTargetsPath)' == '' AND Exists('$(MSBuildExtensionsPath32)\Microsoft\WiX\v3.x\Wix.targets') " />
+  <Target Name="EnsureWixToolsetInstalled" Condition=" '$(WixTargetsImported)' != 'true' ">
+    <Error Text="The WiX Toolset v3.11 (or newer) build tools must be installed to build this project. To download the WiX Toolset, see http://wixtoolset.org/releases/" />
+  </Target>
+  <PropertyGroup>
+    <DefineConstants>target_version=$(target_version);product_name=$(product_name);product_short_name=$(product_short_name);build_dir=buildoutput\;exe_name=$(exe_name);service_args=$(service_args);service=$(service);config_dir=$(config_dir);upgrade_guid=$(upgrade_guid)</DefineConstants>
+  </PropertyGroup>
+  <Target Name="BeforeBuild">
+    <!-- Additional builds can be added here in a similar fashion -->
+	<Exec Command="dotnet publish $(target_project) -c $(target_configuration) -r $(target_runtime) /p:PublishSingleFile=&quot;true&quot; /p:IncludeNativeLibrariesForSelfExtract=&quot;true&quot; /p:InformationalVersion=&quot;$(target_version)&quot; /p:Description=&quot;$(target_desc)&quot; -o buildoutput\" />
```

Comment:
Making this for python could be as simple as replacing this line with the proper build action.